pandas/tests/io/parser/test_dialect.py

```diff
@@ -97,9 +97,9 @@ def test_dialect_conflict_except_delimiter(all_parsers, custom_dialect, arg, val
 
     # arg=None tests when we pass in the dialect without any other arguments.
     if arg is not None:
-        if "value" == "dialect":  # No conflict --> no warning.
+        if value == "dialect":  # No conflict --> no warning.
             kwds[arg] = dialect_kwargs[arg]
-        elif "value" == "default":  # Default --> no warning.
+        elif value == "default":  # Default --> no warning.
```

Comment:
niiiiice

looks like these test cases were always getting skipped previously, so good thing pylint has unearthed this issue - and thanks for fixing it!

Reply:
Sure. I like fixing pylint warnings. Cats are cute by the way :)